### PR TITLE
Create githubvikas-bio.md

### PR DESCRIPTION
**Pull request by chawla1975**
Creating a bio for Vikas Chawla.

This pull request adds a bio for @chawla1975 and needs to be reviewed by @ccadenac

### CODE REVIEW

File: docs/githubvikas-bio.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## @chawla1975
+
+#Vikas Chawla
```
Comment (chawla1975) [inline docs/githubvikas-bio.md:3]: Might need to make italic

File: docs/githubvikas-bio.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+A short bio for *Vikas Chawla*
```
Comment (chawla1975) [inline docs/githubvikas-bio.md]: Please add phone number also

File: docs/githubvikas-bio.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## @chawla1975
+
+#Vikas Chawla
+
+A short bio for *Vikas Chawla (781 354 3086)*
+
+**Hobbies**: Loves to :bike:
```
Comment (chawla1975) [inline docs/githubvikas-bio.md]: please add music also

Comment (chawla1975) [inline docs/githubvikas-bio.md]: sure, will do that

**Comment by matt5r** [comment]
Looks fine. No problem.

**Comment by chawla1975** [comment]
:+1:

**Comment by matt5r** [comment]
:+1:

**Pull request merged**